moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -97,127 +102,60 @@ const handleColumnChange = ({ field }, hidden) => {
   localStorage.setItem("mopedColumnConfig", JSON.stringify(storedConfig));
 };
 
-const useFilterQuery = (locationSearch) =>
-  useMemo(() => {
-    return new URLSearchParams(locationSearch);
-  }, [locationSearch]);
-
 /**
- * if filter exists in url, decodes base64 string and returns as object
- * Used to initialize filter state
- * @return Object
+ * Returns a ProjectStatusBadge component based on the status and phase of project
+ * @param {string} phase - A project's current phase
+ * @param {number} statusId - Project's status id
+ * @return {JSX.Element}
  */
-const useMakeFilterState = (filterQuery) =>
-  useMemo(() => {
-    if (Array.from(filterQuery).length > 0) {
-      try {
-        return JSON.parse(atob(filterQuery.get("filter")));
-      } catch {
-        return {};
-      }
-    }
-    return {};
-  }, [filterQuery]);
+const buildStatusBadge = ({ phaseName, phaseKey }) => (
```

Comment:
Just moving this outside `ProjectListViewTable` since it didn't have any dependencies inside the component.